### PR TITLE
fix(auth): tail logs refresh token #702

### PR DESCRIPTION
**Pull request by proactivebit**
## Description

Add retry mechanism in case of token expired error

## Related Issue

[702](https://github.com/adobe/aio-cli-plugin-cloudmanager/issues/702)

## Motivation and Context

Tail log operation should run with no interraption

## How Has This Been Tested?

Unit tests, regression tests

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: test/cloudmanager-helpers.test.js [hunk 1]
```diff
@@ -133,3 +133,66 @@ test('formatDuration -- only finished', async () => {
...
+  test('should throw error when no 401, 403 thrown', async () => {
+    const mockFn = jest.fn()
+      .mockRejectedValue(new Error())
+
+    await expect(executeWithRetries(mockFn)).rejects.toThrow()
+
+    expect(mockFn.mock.calls.length).toEqual(1)
+  })
+
+  test('should retry 5 times and throw execption', async () => {
```
Comment (sii41383) [inline test/cloudmanager-helpers.test.js:179]: typo `exception`

File: test/commands/environment/tail-log.test.js [hunk 2]
```diff
@@ -47,3 +47,14 @@ test('tail-log - config', async () => {
   await expect(mockSdk.tailLog.mock.calls.length).toEqual(1)
   await expect(mockSdk.tailLog).toHaveBeenCalledWith('5', '17', 'author', 'aemerror', process.stdout)
 })
+
+test('tail-log - should retry 5 times and throw error', async () => {
+  setCurrentOrgId('good')
```
Comment (sii41383) [inline test/commands/environment/tail-log.test.js:52]: Can we mock something other than 'good' ?

Comment (proactivebit) [reply test/commands/environment/tail-log.test.js:52]: I see we use this in other tests

**Review by sii41383 — DISMISSED** [review]
Code, unit test and regression tests look good.

**Review by sii41383 — APPROVED** [review]

**Pull request merged**